treq/auth.py

```diff
@@ -26,6 +27,31 @@ def request(self, method, uri, headers=None, bodyProducer=None):
             method, uri, headers=headers, bodyProducer=bodyProducer)
 
 
+class _PinToFirstHostAgent(object):
+    """
+    An {twisted.web.iweb.IAgent} implementing object that takes two
```

Comment:
"implementing object" is really a content-free phrase here; those words could just be deleted with no loss of meaning.